src/Refactoring-Core/RBTemporaryToInstanceVariableRefactoring.class.st

```diff
@@ -159,10 +176,6 @@ RBTemporaryToInstanceVariableRefactoring >> prepareForExecution [
 RBTemporaryToInstanceVariableRefactoring >> privateTransform [
 
 	self removeTemporaryOfClass: class.
```

Comment:
I do not really undersyand what is removeTemporyOfClass: doing so it is difficult for me to get further. 
Trying to understand the logic: I think that there were something we should keep.
If one of the subclass got the same instance variable that the temp we want to convert, then we remove it from that class and defines it in the root class. 
So why removing this logic. 
Because now if a subclass has the variable with the same name we force the guy to do a pullUp. 
May be this is better this way the developer has to see it.